compositor_render/Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "BUSL-1.1"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
-web_renderer = ["dep:compositor_chromium"]
+web_renderer = ["dep:compositor_chromium", "shared_memory"]
```

Comment:
```suggestion
web_renderer = ["dep:compositor_chromium", "dep:shared_memory"]
```
is this correct?

Reply:
Web renderer works on my branch so I'm assuming that it works 😅. I'll take a closer look into it

Reply:
I'm not sure why it work without `dep:`. I'll add `dep:` anyway for consistency